### ISSUE TITLE
Issue on page /core-concepts - there should be an introduction to approximation

### ISSUE THREAD

**Issue by AndrewGibbs** [ca-knowledgebase/ca-knowledgebase.github.io#30]
There should be an interactive page outlining different approximation approaches, Galerkin, interpolation, etc; with examples of piecewise constants, piecewise linears. This can be explained visually in a single page, but a basic understanding is necessary to use packages like bempp. I think @jahargreaves said he had some nice interactive examples on this already...?

**Comment by jahargreaves** [comment]
Yes I do but they are Matlab Apps. So we'd either need a way to port them to Python or host them as a [Matlab Web App](https://uk.mathworks.com/help/compiler/web-apps.html).
